mods/cnc/languages/rules/en.ftl

```diff
@@ -48,40 +48,47 @@ faction-nod =
     .name = Nod
     .description = Brotherhood of Nod
      The Brotherhood is a religious cult centered around their leader Kane
-     and the alien substance Tiberium. They utilize stealth technology
-     and guerilla tactics to defeat those who oppose them.
+     and the alien substance Tiberium. They use stealth technology
+     and guerrilla tactics to defeat those who oppose them.
 
 ## defaults.yaml
-notification-unit-lost = Unit lost.
-notification-unit-promoted = Unit promoted.
-notification-building-captured = Building captured.
-notification-civilian-building-captured = Civilian Building captured.
-notification-structure-lost = Structure lost.
+notification-unit-lost = Unit lost
+notification-unit-promoted = Unit promoted
+notification-building-captured = Building captured
+notification-civ-building-captured = Civilian building captured
+notification-structure-lost = Structure lost
 
 ## aircraft.yaml
 actor-tran =
    .name = Chinook Transport
-   .description = Fast Infantry Transport Helicopter.
+   .description = Fast Infantry Transport Helicopter
       Unarmed
+   .encyclopedia = The chinook is a flying transport capable of holding ten infantry. In general, it is used to transport commandos or engineers for backline-destruction.
+    \Like all aircraft, it can be handy in a pinch to crush a commando by landing! (alt)
```

Comment:
I would remove the hotkey reference, since the player may have remapped the hotkey. (Unless this text is hotkey aware?)